src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct

```diff
@@ -231,6 +231,17 @@
             -->
 
       <!-- Text editor commands -->
+      <!-- TODO: Remove once https://github.com/dotnet/roslyn/issues/8927 is done. -->
+      <Button guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" priority="0x101" type="Button">
+        <Parent guid="guidInteractiveWindowCmdSet" id="TextEditorGroup" />
+        <CommandFlag>DynamicVisibility</CommandFlag>
+        <CommandFlag>DefaultInvisible</CommandFlag>
+        <Strings>
+          <ButtonText>Execute in Interactive</ButtonText>
```

Comment:
That's right , checkin deadline for last localization prior to escrow is 
​6pm Sunday Mar 6, that would be the build used for last hand off on 3/7 (d14rel)